zbus_macros/src/lib.rs

```diff
@@ -189,6 +189,23 @@ mod utils;
 ///
 /// [`zbus_polkit`] is a good example of how to bind a real D-Bus API.
 ///
+/// # Custom crate path
+///
+/// If you've renamed `zbus` in your `Cargo.toml` or are using it through a re-export,
+/// you can specify the crate path using the `crate` attribute:
+///
+/// ```
+/// use zbus_macros::proxy;
```

Comment:
We don't want people to use the macros and derive crates directly so the docs shouldn't use them either. This is why zbus is the dev-dep of zbus_macros and similarly for zvariant and zm.